api/auth/signin.js

```diff
@@ -1,13 +1,12 @@
+//Signin route
 const express = require('express')
 const database = require('../models/user.model')
 const route = express.Router();
 
-
-
-
 route.post('/signin', async(req, res)=>{
     const {username, email, password} = req.body
-    let match = false
+     let match = false
+//loop through database and check if if email && password === database.users[i].email && database.users[i].password
 for(let i =0; i < database.users.length; i++){
```

Comment:
@Adegbite1999 there's a better way of handling this. You can do `database.users.find((user) => email === user.email && password === user.password)`. it's more efficient.
Read up about Array.some and Array.find and Array.filter.